### PR TITLE
Add handcalcs

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/handcalcs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - flit
+  run:
+    - innerscope >=0.2.0
+    - jupyterlab >=3.0.0
```
Comment (dopplershift) [inline recipes/handcalcs/meta.yaml:25]: Upstream only lists this as a "dev" dependency? Is that incorrect?

**Review by dopplershift — APPROVED** [review]
Recipe looks good. Just had a question.

File: recipes/handcalcs/meta.yaml (hunk 1 above)
Comment (jan-janssen) [reply recipes/handcalcs/meta.yaml]: I used grayskull to create the recipe - but you are right it should work without `jupyterlab` - I am testing it right now.

**Pull request merged**